src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const total = items.length;
+
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState('5');
```

Comment:
```suggestion
  const [perPage, setPerPage] = useState(5);
```
it can be the number